.gitignore

```diff
@@ -5,3 +5,4 @@ config.json
 .vscode/
 magic-mirror.db
 coverage/
+values.yaml
```

Comment:
Does this also end up ignoring `values.yaml` in the `helm` directory?

Reply:
No. It's relative to the root directory.